src/schema/sale/index.js

```diff
@@ -294,17 +214,10 @@ export const SaleType = new GraphQLObjectType({
             return PREDICTION_ENDPOINT + "/" + sale.id
           }
         },
-      },
-      // Only fetches the partner info that's already included in the Sale object
+      }, // Only fetches the partner info that's already included in the Sale object
       // since we don't (at this time) need to load the full Partner object.
-      partner: {
-        type: Partner.type,
-        resolve: ({ partner }) => partner,
-      },
-      profile: {
-        type: Profile.type,
-        resolve: ({ profile }) => profile,
-      },
+      partner: { type: Partner.type, resolve: ({ partner }) => partner },
+      profile: { type: Profile.type, resolve: ({ profile }) => profile },
```

Comment:
@mzikherman - i think something is up with your Editor / new comp. Prettier shouldn't be reformatting this stuff.